packages/lib/src/interactions/SelectionTool.tsx

```diff
@@ -50,84 +52,76 @@ function SelectionTool(props: Props) {
     disabled,
   });
 
+  const computeSelection = useCallback(
+    (worldEndPt: Vector3) => {
+      assertDefined(startEvt);
+      const { dataPt: startPoint, worldPt: worldStartPoint } = startEvt;
+      const boundWorldEndPoint = boundPointToFOV(worldEndPt, camera);
+
+      return {
+        startPoint,
+        endPoint: worldToData(boundWorldEndPoint),
+        worldStartPoint: new Vector2(worldStartPoint.x, worldStartPoint.y),
```

Comment:
I can now use `worldPt` almost directly -- just need to convert it to `Vector2`. I have an idea to fix this for another PR.